eirh_subtheme.libraries.yml

```diff
@@ -94,8 +94,13 @@ allpages:
 
 eventshub:
   version: VERSION
+  header: false
   css:
     theme:
       src/eventshub/dist/index.css: {}
   js:
-    src/eventshub/dist/eventshub.js: {}
+    src/eventshub/dist/eventshub.js:
+      attributes:
+        async: true
+      preprocess: false
```

Comment:
Remove from the preprocessed group because that file can take 5 - 8 seconds to load.